pkg/sdk/logging/model/syslogng/output/tls.go

```diff
@@ -41,7 +41,7 @@ type TLS struct {
 	// Name of a file, that contains an X.509 certificate (or a certificate chain) in PEM format, suitable as a TLS certificate, matching the private key set in the key-file() option. [more information](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.37/administration-guide/73#kanchor3146)
 	CertFile *secret.Secret `json:"cert_file,omitempty"`
 	// Verification method of the peer. [more information](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.37/administration-guide/73#tls-options-peer-verify)
-	PeerVerify string `json:"peer_verify,omitempty"`
+	PeerVerify *bool `json:"peer_verify,omitempty"`
```

Comment:
wasn't `peer_verify` (with underscore) invalid?

Reply:
actually it's valid, i was just misinterpreting the error message.

Reply:
but if we want, we can still change this field to use a dash, but i didn't want to introduce unnecessary (breaking) api changes.

Reply:
I would say that we should leave it as is, because both the underscore and dash versions can be found in the codebase now, and once there is a an agreed preference then we should do big formatting PR.